docs/documentation/actions/amqp.md

```diff
@@ -1,90 +1,131 @@
+!!! important "Target Configuration"
+    For all actions, a AMQP target should be defined and have a `name` and a valid `url`.
+
+```json title="Rabbitmq target"
+{
+    "name":"RABBITMQ_TARGET",
+    "url":"amqp://localhost:5672",
+    "properties": [
```

Comment:
ad you do for kafka, i would like to have a target with ssl configuration as example